module/VuFind/src/VuFind/Db/Service/UserListService.php

```diff
@@ -221,17 +240,25 @@ public function getUserListsByTagAndId(
      * Get list objects belonging to the specified user.
      *
      * @param UserEntityInterface|int $userOrId User entity object or ID
+     * @param string                  $types    Types of user lists to get from database.
+     *                                          Use UserList::TYPE_ALL to get user lists
+     *                                          without checking types.
```

Comment:
This can be unwrapped:

```suggestion
     * Use UserList::TYPE_ALL to get user lists without checking types.
```